__tests__/app.test.js

```diff
@@ -24,4 +39,15 @@ describe("GET /api/topics", () => {
                 })
             })
     })
+})
+
+describe("invalid route", () => {
+    test("404: Error - responds with an error when the route is not exist", () => {
+        return request(app)
+            .get("/books")
```

Comment:
Just to be extra safe and future proof this test, instead of `/books` you could be more explicit and put `/not-an-endpoint` or similar. Also I would include the `/api` before it too as it is the api we are testing here.